### PR TITLE
systemvm: fix proc.find in CsProcess.py

### PR DESCRIPTION
**Pull request by weizhouapache**
## Description

This fixes the issues described in #4384 and #4386 

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

### CODE REVIEW

File: systemvm/debian/opt/cloud/bin/cs/CsProcess.py [hunk 1]
```diff
@@ -40,9 +40,9 @@ def kill_all(self):
 
     def find_pid(self):
         self.pid = []
+        items = len(self.search)
         for i in CsHelper.execute("ps aux"):
-            items = len(self.search)
-            proc = re.split(r"\s+", i)[items*-1:]
+            proc = re.split(r"\s+", i)[10:]
```
Comment (DaanHoogland) [inline systemvm/debian/opt/cloud/bin/cs/CsProcess.py:45]: given this is null based the return is now the full cli as returned by `ps aux` ; :+1:

File: systemvm/debian/opt/cloud/bin/cs/CsProcess.py [hunk 2]
```diff
@@ -40,9 +40,9 @@ def kill_all(self):
 
     def find_pid(self):
         self.pid = []
+        items = len(self.search)
```
Comment (DaanHoogland) [inline systemvm/debian/opt/cloud/bin/cs/CsProcess.py:43]: maybe we should think about renaming search; it actually means to be the process/command name.

**Review by DaanHoogland — APPROVED** [review]
given good test results LGTM

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by DaanHoogland** [comment]
ping @rhtyd , not sure who else was involved in the predecessors of this one, but i think we can merge this one now.

**Comment by DaanHoogland** [comment]
@davidjumani @borisstoyanov @andrijapanicsb @GabrielBrascher can we get more oks on this?

**Review by rhtyd — APPROVED** [review]

**Comment by weizhouapache** [comment]
@DaanHoogland good .
let's see if there are still issues with test_vpc_redundant.py in trillian test of coming PRs.

**Comment by weizhouapache** [comment]
@DaanHoogland this has been merged into 4.14. it needs to be merged into master as well

**Pull request merged**